MatrixSDK/Categories/MXRestClient+Extensions.swift

```diff
@@ -35,7 +35,7 @@ public extension MXRestClient {
                              failure: @escaping (_ error: NSError?) -> Void) -> MXHTTPOperation {
         
         // Do not chunk if not needed
-        if users.count < chunkSize {
+        if users.count <= chunkSize {
```

Comment:
Naughty naughty. Will write a test for this in a separate PR as it requires some integration test stubbing